core-concepts/did-ddo.md

```diff
@@ -313,6 +313,8 @@ An asset with a service of `type` `compute` has the following additional attribu
 | `boolean`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | **✓**    | If `true`, any passed raw text will be allowed to run. Useful for an algorithm drag & drop use case, but increases risk of data escape through malicious user input. Should be `false` by default in all implementations. |
 | Type                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Required | Description                                                                                                                                                                                                               |
 | `boolean`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | **✓**    | If `true`, any passed raw text will be allowed to run. Useful for an algorithm drag & drop use case, but increases risk of data escape through malicious user input. Should be `false` by default in all implementations. |
+| Type                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Required | Description                                                                                                                                                                                                               |
```

Comment:
This change is not intentional. It is created by gitbook bot!

Reply:
That's weird